tests/cases/standalone/common/flow/flow_call_df_func.result

```diff
@@ -224,13 +226,13 @@ admin flush_flow('test_numbers_df_func');
 | 1                                        |
 +------------------------------------------+
 
-SELECT col_0, col_1 FROM out_num_cnt_df_func;
+SELECT maxmin, time_window FROM out_num_cnt_df_func;
 
-+-------+---------------------+
-| col_0 | col_1               |
-+-------+---------------------+
-| 2     | 2021-07-01T00:00:00 |
-+-------+---------------------+
++--------+---------------------+
+| maxmin | time_window         |
```

Comment:
Is this a breaking change or it's compatible with existing tables

Reply:
> Is this a breaking change or it's compatible with existing tables

compatible, flow will always use existing table info if exist, and only fallback to auto create table when table name and id can't be acquire from `TableSource` manager